release/linux-specific/OmegaT

```diff
@@ -7,4 +7,4 @@ JAVA="java"
 BUNDLED_JAVA="${REALOMEGATPATH}/jre/bin/java"
 [ -f "${BUNDLED_JAVA}" ] && JAVA="${BUNDLED_JAVA}"
 
-"${JAVA}" -jar -Xmx1024M "${REALOMEGATPATH}/@JAR_SUBST@" "$@"
+"${JAVA}" -Djava.system.class.loader=org.omegat.MainClassLoader -jar -Xmx1024M "${REALOMEGATPATH}/@JAR_SUBST@" "$@"
```

Comment:
Question.

Where is a place to do  a same thing in macOS ?

Reply:
Probably here:

https://github.com/omegat-org/omegat/blob/6aab48baa3b1c036fb66e75c1eba80ce41ff40e2/release/mac-specific/OmegaT.app/Contents/Info.plist#L81-L83

If you can supply a JAR of your LAF plugin I can test it.

Reply:
I tested your sample plugin and it worked: when launching the OmegaT.app, the theme combobox showed FlatLaf (light and dark) and changing the theme took effect after relaunching the app.